CHANGELOG.md

```diff
@@ -25,7 +25,9 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 - ROX-10038: Removed limit of 10 inclusions and 10 exclusions from policy form
 - ROX-10090: Made the username and password optional on the Artifactory integration form
 - ROX-10217: Remove format validation from the URL field of the generic webhook integration form
-
+- ROX-10270, ROX-10272: In 71, we plan to deprecate the following:
+  1. The field name `categories` in the policy object. This will be replaced with a new `policyCategories` field.
```

Comment:
Is it just a rename? If not, you should specify the structure of the expected field.

Reply:
Structure will be implemented in next PR, will update then if required. Not just a rename - old field was a list of name strings, new field is a list of uuids.

Reply:
If the structure of the new field is not settled, then why rush to send out an incomplete deprecation. It defeats the purpose of advance notice. You are informing customers about the deprecation leaving a mystery about how should the customers use the new field.

Reply:
Disagree, it will be updated as needed. With 9 week release cadence and our requirement about heads up, this is better done sooner. Past experience also shows the same.

Reply:
Also: For customers it will be documented - the new APIs et al. And the new policy field should not change any behavior for customers from a UI standpoint.

Reply:
Then what is the point of advance notice if customers have to wait for the release of the feature to determine the workaround? Note that the planned change is a breaking change.

Reply:
Remove this from deprecation and instead as a top level:

```suggestion
  Policy field `categories` will be decoupled from policy definition. Categories will be moved to another top-level object. This field in policies will be derived from the new object and thus if `categories` are provided in a POST or PUT API request, it will be ignored. Categories in policy imports, searches, notifications and roxctl will continue to work as-is. If this is problematic, please reach out to support.
```

Something like that?

Reply:
The suggestion reads better.

Reply:
`Categories in policy imports, searches, notifications and roxctl will continue to work as-is. If this is problematic, please reach out to support.` -> Not sure about the wording of this, but don't have a better suggestion. 

What I would like to convey to the customers: Please stop using or relying on the "categories" field. In any way. This allows us to deprecate it easily in the future. The above suggested text is not bringing then out. But as I said, no better suggestion yet either. Will think.

Reply:
Another suggestion: Call it deprecation, so customers stop relying on it (doesn't look like they are anyway).  Even if that means we populate it in certain paths. That leaves the possibility of a full deprecation, open.

Reply:
Hopefully @kcarmichael08 can help with wording. 

@clickboo maybe: 
```Policy field `categories` will be decoupled from policy definition. Categories will be moved to another top-level object. This field in policies will be derived from the new object. The `categories` field in the policy is object is no longer the source of truth. For example, if the field is provided in a POST or PUT API request, it will be ignored.```

something like that?

Reply:
`and maybe deprecated in future releases`?